src/main/tut/index.html

```diff
@@ -99,12 +101,16 @@
 
 --
 
-Better, but not great: you have to explicitly specify how to decode types every time.
+<!-- COMMENT -->
+
+Better, but not great: one has have to manually convert the value to the desired type every time.
 
 ---
 
 ## Small detour: implicits
 
+<!-- COMMENT: using the word implicit to describe implicits seems sub-optiomal? -->
+
 > When a function expects a parameter of type `T` _and_ that parameter is marked as implicit _and_ there exists a
 > value of type `T` marked as implicit in scope, then the compiler can use that value automatically.
```

Comment:
Never thought about it that way, but you're absolutely right and that paragraph now bothers me quite a lot. Thanks a lot!